File: sources/server/server/Hotel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace server
{
    public class Hotel
    {
        public int Hid { get; set; }
        public String Name { get; set; }
        public String Adr { get; set; }

        public Hotel() { }

        public Hotel(int Hid, String Name, String Adr)
        {
            this.Hid = Hid;
            this.Name = Name;
            this.Adr = Adr;
        }
    }
}

File: sources/customer/src/com/philippspiess/hotel/api/model/Hotel.java
package com.philippspiess.hotel.api.model;

public class Hotel {
	public int Hid;
	public String Name;
	public String Adr;
	
	
	public int getHid() {
		return Hid;
	}
	public void setHid(int hid) {
		Hid = hid;
	}
	public String getName() {
		return Name;
	}
	public void setName(String name) {
		Name = name;
	}
	public String getAdr() {
		return Adr;
	}
	public void setAdr(String adr) {
		Adr = adr;
	}	
	
	public String toString() {
		return Name + " - " + Adr;
	}
}
